### DISCUSSION TITLE
Hash

### DISCUSSION THREAD

**Discussion by Keystarmusic** [blockscout/blockscout#5447] [Support]
Blockscout not enable to locate transaction hash on omni bridge. I need to approve smart contract because once I approved with wrong address.  Now I must make it again but for that I need I think to locate transaction hash on blockscout but it's not find it. What me do.

**Comment by igorbarinov** [comment]
Please provide links to transactions in question

**Comment by Keystarmusic** [comment]
This is the contract for booth transactions which exchanges don't except. If we can make it to this original contract of ecop 0x0106a1122fe94a9cf151097c1fe17229ec78ffad I will be happy. Thank you!

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:02, Igor ***@***.***> wrote:   


Please provide links to transactions in question

**Comment by Keystarmusic** [comment]
I forget to told you that I copy this contract on from blockscout and coins now appear in my wallet

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:30, David ***@***.***> wrote:   This is the contract for booth transactions which exchanges don't except. If we can make it to this original contract of ecop 0x0106a1122fe94a9cf151097c1fe17229ec78ffad I will be happy. Thank you!

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:02, Igor ***@***.***> wrote:   


Please provide links to transactions in question

**Comment by Keystarmusic** [comment]
But also they stay on omni bridge interface

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:30, David ***@***.***> wrote:   This is the contract for booth transactions which exchanges don't except. If we can make it to this original contract of ecop 0x0106a1122fe94a9cf151097c1fe17229ec78ffad I will be happy. Thank you!

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:02, Igor ***@***.***> wrote:   


Please provide links to transactions in question

**Comment by Keystarmusic** [comment]
Can you help me

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:36, David ***@***.***> wrote:   But also they stay on omni bridge interface

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:30, David ***@***.***> wrote:   This is the contract for booth transactions which exchanges don't except. If we can make it to this original contract of ecop 0x0106a1122fe94a9cf151097c1fe17229ec78ffad I will be happy. Thank you!

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:02, Igor ***@***.***> wrote:   


Please provide links to transactions in question

**Comment by Keystarmusic** [comment]
I think is not right to change contract when bridging I think it's must stay same why than it's bridge when it not work. You must help me now to change contract from bridging to original contract to ecop.

Sent from Yahoo Mail on Android 
 
  On пн, апр 25, 2022 at 21:30, David ***@***.***> wrote:   Can you help me

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:36, David ***@***.***> wrote:   But also they stay on omni bridge interface

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:30, David ***@***.***> wrote:   This is the contract for booth transactions which exchanges don't except. If we can make it to this original contract of ecop 0x0106a1122fe94a9cf151097c1fe17229ec78ffad I will be happy. Thank you!

Sent from Yahoo Mail on Android 
 
  On нд, апр 24, 2022 at 13:02, Igor ***@***.***> wrote:   


Please provide links to transactions in question
